src/pgcopy/src/copy.rs

```diff
@@ -60,17 +61,28 @@ pub fn encode_copy_row_binary(
     Ok(())
 }
 
+/// Encodes given `row` into bytes with default param values
+/// in `CopyTextFormatParams`.
+pub fn encode_copy_row_text_default(
+    row: Row,
+    typ: &RelationType,
+    out: &mut Vec<u8>,
+) -> Result<(), io::Error> {
```

Comment:
I don't think this method is carrying its weight as it's only called in one place. Can you just adjust the pgwire protocol module to pass in `CopyTextFormatParas::default()` instead?

Reply:
Removed!